core/src/main/java/hivemall/optimizer/EtaEstimator.java

```diff
@@ -157,4 +158,34 @@ public static EtaEstimator get(@Nullable CommandLine cl, float defaultEta0)
         return new InvscalingEtaEstimator(eta0, power_t);
     }
 
+    @Nonnull
+    public static EtaEstimator get(@Nonnull final Map<String, String> options)
+            throws IllegalArgumentException {
+        final String etaName = options.get("eta");
+        if(etaName == null) {
+            return new FixedEtaEstimator(1.f);
```

Comment:
I think default initial eta value can be same to the default `eta0` value (0.1f). 1.f looks large.